index.bs

```diff
@@ -2898,3 +2903,7 @@ To decode a PSK `P` given a QR code, follow these steps:
 
 1. Obtain the string `N` by decoding the QR code.
 2. Parse `N` as a hexadecimal number to obtain `P`.
+
+Appendix D: Entire Flow Chart {#appendix-d}
+=============================
+<img src="./images/entire_flow_chart.svg" alt="entire flow chart">
```

Comment:
```suggestion
<img src="./images/entire_flow_chart.svg" alt="Entire flow chart" style="width: 100%; height: 100%">
```

This will fix the following markup validator error:
`error: Bad value “100%” for attribute “height” on element “img”: Expected a digit but saw “%” instead.`

Reply:
Thanks anssi for the suggestion, I have changed this.